scripts/falcoctl/falcoctl.yaml.in

```diff
@@ -1,3 +1,10 @@
+driver:
+    type: "kmod"
+    name: "@DRIVER_NAME@"
+    repos:
+        - "@DRIVERS_REPO@"
+    version: "@DRIVER_VERSION@"
+    hostroot: ""
```

Comment:
This may be misleading. In the container context, this should be defaulted to `/host`.

Reply:
How can we know if we will run in a container context? I think we will need to pass FALCOCTL_DRIVER_HOSTROOT env variable for that.

Reply:
I will update `falcoctl` PR to check `HOST_ROOT` env variable (that is used also by Falco and libs and we cannot change its name to `FALCOCTL_DRIVER_HOSTROOT` for host root.

Reply:
Ok, i let falcoctl properly manage "HOST_ROOT" env var (binding to "FALCOCTL_DRIVER_HOSTROOT") for us; therefore this should not be a problem anymore since all the docker files set HOST_ROOT env variable: 
```
ENV HOST_ROOT /host
```